test/zIntegrationTests.ts

```diff
@@ -127,3 +162,76 @@ describe("Test suit for setting up the stage", function () {
         expect(await r.leverageAllocator.getAddressToLvUSDAvailable(await user.getAddress())).to.equal(parseUnitsNum(initialUserLevAllocation));
     });
 });
+
+describe("Test suit for getting leverage", function () {
+    let leverageUserIsTaking: number;
+
+    before(async function () {
+        await setupEnvForIntegrationTests();
+        leverageUserIsTaking = getFloatFromBigNum(
+            await r.parameterStore.getAllowedLeverageForPosition(userOUSDPrincipleInEighteenDecimal, numberOfCycles));
+        console.log("Will take %s lev while user has allocation of %s", leverageUserIsTaking,
+            getFloatFromBigNum(await r.leverageAllocator.getAddressToLvUSDAvailable(user.address)));
+
+        await approveAndGetLeverageAsUser(userOUSDPrincipleInEighteenDecimal, numberOfCycles, r, user);
+        positionId = 0;
+    });
+
+    it("Should have created a single position and assign it to user", async function () {
+        printPositionState(r, positionId);
+        printPoolState(r.curveLvUSDPool);
+        printMiscInfo(r, user);
+        const nftBalance = await r.positionToken.balanceOf(user.address);
```

Comment:
nftBalance is the number of positions an address has?

Reply:
Yeah